src/views/History.vue

```diff
@@ -1,41 +1,185 @@
 <template>
-  <div class="container">
-    <page-header title="Trade History">
-      <div class="breadcrumps" slot="breadcrumps">
-        <router-link to="/" class="active">{{ $t('views.history.home') }}</router-link>
-        {{ $t('views.history.') }}
-        <router-link to="/browse" class="current">{{ $t('views.history.trade_history') }}</router-link>
+  <div class="faucet">
+    <div class="faucet-content">
+      <div class="container mb-5 column py-3 p-3 d-flex">
+        <h1>{{ $t('views.history.trade_history') }}</h1>
+        <div class="py-3">
+          <div v-if="showHistoryTable"><faucet-table :items="history"></faucet-table></div>
+          <div v-else>
+            <h5>
+              You haven't made any trades yet
```

Comment:
should say no activity detected in last x number of blocks, and make a link to go back further in time